modules/cnv_template_import_dv.adoc

```diff
@@ -0,0 +1,22 @@
+[[template-import-dv]]
+=== Template: DV Import Configuration File
+
+*example-import-dv.yaml*
+
+----
+apiVersion: cdi.kubevirt.io/v1alpha1
+kind: DataVolume
+metadata:
+  name: "example-import-dv"
+spec:
+  source:
+      http:
+         url: "https://download.fedoraproject.org/pub/fedora/linux/releases/28/Cloud/x86_64/images/Fedora-Cloud-Base-28-1.1.x86_64.qcow2" # Or S3
+         secretRef: "" # Optional
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: "64Mi"
```

Comment:
This would not be large enough to accommodate the download.  Change to 1G.

Reply:
Fixed: https://github.com/openshift/openshift-docs/pull/12518/commits/8f994e43c293c237fe3c2192c2f6aa220c163b8c